src/sfdt/processInlines.ts

```diff
@@ -1,5 +1,4 @@
-import get from 'lodash/get';
-import filter from 'lodash/filter';
+import {get, filter, forEach} from 'lodash';
```

Comment:
can you please use individual import statements as before, it helps to reduce the bundle size. Thx